### ISSUE TITLE
✅ 🧩 🐞 The Advanced tab in the Widget Panel appears empty when the Optimized Markup experiment and Happy Addons are activated

### ISSUE THREAD

**Issue by djohn151** [elementor/elementor#29460]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [ ] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

The advanced tab in the editor is empty when Optimized Markup is activated.

### Steps to reproduce

Activate Optimized Markup in elementor beta and go to the advanced tab in the editor for a widget

### Expected behavior

The advanced tab in the editor is empty when Optimized Markup is activated.

### Isolating the problem

- [x] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

```txt

== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Ubuntu 22.04 v10.6.18
	PHP Version: 8.3.12-nmm1
	PHP Memory Limit: 768M
	PHP Max Input Vars: 6000
	PHP Max Post Size: 768M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.7.1
	Site URL: https://www.xxx.de
	Home URL: https://www.xxx.de
	WP Multisite: No
	Max Upload Size: 768 MB
	Memory limit: 768M
	Max Memory limit: 768M
	Permalink Structure: /%category%/%postname%/
	Language: de_DE
	Timezone: Europe/Berlin
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor Theme
	Version: 1.0.4
	Author: 
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 3.1.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: de
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36

== Active Plugins ==

	Contact Form 7
		Version: 6.0.1
		Author: Takayuki Miyoshi

	Elementor
		Version: 3.26.0-beta2
		Author: Elementor.com

	Elementor Pro
		Version: 3.26.0-beta2
		Author: Elementor.com

	Happy Elementor Addons
		Version: 3.14.1
		Author: Leevio

	Happy Elementor Addons Pro
		Version: 2.15.1
		Author: Leevio


== Eigenschaften ==
	Custom Fonts: 2
	Custom Icons: 0

== Integrations ==
	


== Elementor-Experimente ==
	Inline-Schriftarten-Symbole: Aktiv
	Additional Custom Breakpoints: Standardmäßig aktiviert
	Container: Aktiv
	Upgrade Swiper Library: Standardmäßig aktiviert
	Nested Elements Performance: Standardmäßig aktiviert
	Optimized Markup: Aktiv
	Plugin Onboarding: Standardmäßig aktiviert
	CSS Smooth Scroll: Standardmäßig aktiviert
	Default to New Theme Builder: Inaktiv
	Header und Footer: Inaktiv
	Elementor Home Screen: Standardmäßig aktiviert
	Startseiten: Aktiv
	Nested Elements: Aktiv
	Editor Top Bar: Inaktiv
	Element Caching: Inaktiv
	Link In Bio: Standardmäßig aktiviert
	Floating Buttons: Standardmäßig aktiviert
	Elementor Editor Events: Standardmäßig deaktiviert
	Atomic Widgets: Standardmäßig deaktiviert
	Global Classes: Standardmäßig deaktiviert
	Launchpad Checklist: Standardmäßig deaktiviert
	Menü: Inaktiv


== Protokoll ==
	
[redacted by moderation] 



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Kompatibel
	Happy Elementor Addons: Unkompatibel
	Happy Elementor Addons Pro: Unkompatibel

== Elementor Pro - Compatibility Tag ==
	
	Happy Elementor Addons: Unkompatibel
	Happy Elementor Addons Pro: Unkompatibel



```
</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hi @djohn151!

Thank you for your feedback!

What you are reporting is caused by the Happy Addons plugin, which is incompatible with the Optimized Markup experimental feature.

We have already contacted the developers of this add-on to help them make their plugins compatible, but we will reinforce the message. We recommend you doing the same.

When this add-on is deactivated the Advanced tab appears as expected.

Before activating experimental features, we advise ensuring any add-on installed claims compatibility with them.

Nonetheless, thank you for taking the time to share your experience with us.

Kind regards

**Comment by sayedulsayem** [comment]
Hi @djohn151 

This is Sayem from the Happy Addons team. I will take over this issue in our system. I will try to address it ASAP. You are free to open this issue in our support system or our GitHub to get updated.

Here is the link: https://github.com/team-leevio/happy-elementor-addons/issues

@nicholaszein I appreciate your insight.

Thanks

**Comment by sayedulsayem** [comment]
hi @djohn151 

I have tested it according to your instructions. I can't reproduce the issue, but it is working every time. I think this is not a straightforward issue.

Please, open this issue in our support system or our GitHub issues to work on this further. 

Thanks

**Comment by djohn151** [comment]
Hello @sayedulsayem.
I have described the problem here and that should be sufficient. I don't have time for this.
Thank you very much.

**Comment by sayedulsayem** [comment]
Hi @djohn151 
I tried reproducing the issue as described, but I was unable to. However, it is working on my test.
If you don't cooperate to reproduce the issue, I won't be able to fix that. 
Thanks

**Comment by djohn151** [comment]
@nicholaszein I have now deactivated all plugins except elementor and elementor pro and the problem still exists.

**Comment by artkrsk** [comment]
> We have already contacted the developers of this add-on to help them make their plugins compatible, but we will reinforce the message. We recommend you doing the same.
@nicholaszein can you please share a fix for developers here?

Yesterday I started to implement compatibility with the newest "Optimized Markup" feature for my widgets. I've added the following public method to my widget class:
```php
<?php

class Backgrounds_Slider_Widget extends Asli_Widget_Base {
  public function has_widget_inner_wrapper(): bool {
    return false;
  }
}
```
The inner wrapper was gone and the widget displayed normally. However I wasn't able to open "Advanced" tab of the widget. And there is JS error in console:
```
editor-modules.min.js?ver=3.26.0-beta3:3 Uncaught TypeError: Cannot read properties of undefined (reading 'cid')
    at Module.getEditorControlView (editor-modules.min.js?ver=3.26.0-beta3:3:22022)
    at T.each.L.<computed> [as getEditorControlView] (common-modules.min.js?ver=3.26.0-beta3:2:58634)
    at a.Channel.highlightIconIfFilled (display-conditions.min.js?ver=3.25.3:2:47599)
    at p (load-scripts.php?c=1&load%5Bchunk_0%5D=underscore,jquery-core,jquery-migrate,backbone,jquery-ui-core,jquery-ui-mouse,wp-api-request,wp-hooks&ver=6.7.1:10:3758)
    at f (load-scripts.php?c=1&load%5Bchunk_0%5D=underscore,jquery-core,jquery-migrate,backbone,jquery-ui-core,jquery-ui-mouse,wp-api-request,wp-hooks&ver=6.7.1:10:3497)
    at a (load-scripts.php?c=1&load%5Bchunk_0%5D=underscore,jquery-core,jquery-migrate,backbone,jquery-ui-core,jquery-ui-mouse,wp-api-request,wp-hooks&ver=6.7.1:10:692)
    at e.trigger (load-scripts.php?c=1&load%5Bchunk_0%5D=underscore,jquery-core,jquery-migrate,backbone,jquery-ui-core,jquery-ui-mouse,wp-api-request,wp-hooks&ver=6.7.1:10:3380)
    at n.openActiveSection (editor.min.js?ver=3.26.0-beta3:3:723277)
    at n.onRenderCollection (editor-modules.min.js?ver=3.26.0-beta3:3:27803)
    at Object._triggerMethod (backbone.marionette.min.js?ver=2.4.5.e1:24:7999)
```

Note: if I opt-out from the feature by returning `false` in `has_widget_inner_wrapper()`, the tab starts to open as it should without errors.

**Comment by djohn151** [comment]
The error only occurs with widgets that have the "elementor-widget-container" class. With containers, the expanded tab is displayed normally

**Comment by nicholaszein** [comment]
Hi everyone!

We have great news! 🙌 

### 📢 We're happy to announce that this issue was resolved with the release of Elementor v3.26.0! 🥳 

After some investigation we were able to identify the problems that were causing this issue to happen, and we fixed it to the best of our abilities. However, since the **Optimized Markup** Experimental Feature is still in the _**alpha**_ phase, it is not impossible that we might encounter unforeseen problems with this feature. If you are still having trouble after updating to Elementor v3.26.0, please open a new issue with details so our team can take another look and find additional solutions.

Thank you for the feedback! 🙏 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
